en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+# Create a REST API
+
+A REST API is an Application Programming Interface that conforms to the constraints of REST architectural styles and principles. REST is an [architectural style](https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) introduced by Dr. Roy Fielding in his doctoral dissertation in the year 2000. It was initially designed as a guide for the development of the web. Today, REST is being widely adopted for the creation of high-performance, stateless and reliable APIs that are used by numerous applications.
+
+Choreo’s low-code editor allows developers to easily design (and then implement) high-quality REST APIs. To explore this capability, let's consider a scenario where an Analyst needs to retrieve the daily count of COVID-19 patients per one million population by country. In this tutorial, you will create an API that supports this.
+
+## Develop
+
+In this section, let's develop the application that retrieves COVID-19 related statistics.
+
+### Create a project and add a REST API component
+
+1. Sign in to the Choreo Console at https://consolev2.preview-dv.choreo.dev/.
```

Comment:
Changed it for now, but we had it in this format in V1. The reason may be because then the user can copy the URL more easily to open it in an incognito tab. Let me check on this.